jsweb/slides/dom.html

```diff
@@ -1,444 +1,321 @@
- &lt;head&gt;
-  &lt;meta charset=&quot;utf-8&quot;&gt;
-  &lt;title&gt;All About Cats&lt;/title&gt;
-  &lt;style type=&quot;text/css&quot;&gt;
-   h1 {
-     color: red;
-   }
-  #mainpicture {
-    border: 1px solid black;
-  }
-  .catname {
-    font-weight: bold;
-  }
-  &lt;/style&gt;
- &lt;/head&gt;
- &lt;body&gt;
-  &lt;h1&gt;CATS!&lt;/h1&gt;
-  &lt;img id=&quot;mainpicture&quot; src=&quot;http://placekitten.com/200/300&quot;&gt;
-  &lt;p&gt;So cute!&lt;/p&gt;
-  &lt;ul&gt;
-   &lt;li class=&quot;catname&quot;&gt;Lizzie&lt;/li&gt;
-   &lt;li class=&quot;catname&quot;&gt;Daemon&lt;/li&gt;
-  &lt;/ul&gt;
- &lt;/body&gt;
-&lt;/html&gt;
-</pre>
-     </section>
+      <header><h1>Review: HTML &amp; CSS</h1></header>
+      <br/>
+      <section class="content">
+        <a class="jsbin-embed" href="http://jsbin.com/mupemodeqa/embed?html,css,output">JS Bin on jsbin.com</a><script src="http://static.jsbin.com/js/embed.min.js?3.36.16"></script>
+        <p>To open this JS Bin in a new window, click <a href="http://jsbin.com/mupemodeqa/1/edit?html,css,output">here</a>.</p>
+      </section>
     </div>
-
-
     <div class="slide normal">
-     <header><h1>HTML Editors</h1></header>
-     <section class="content">
-     <section class="content">
-     <p>Since HTML files are just text files, many programs can be used to create them. Some programs provide special assistance for handling HTML, like syntax-highlighting or autocompletion.
-</p>
-<table style="font-size:120%; border: 1px solid black;" border="1" cellspacing="0">
- <thead>
-  <tr><th></th><th>Windows</th><th>Mac</th><th>Online</th></tr>
- </thead>
-  <tbody>
-  <tr><td>Free</td>
-        <td><a href="http://notepad-plus-plus.org/">Notepad++</a></td>
-       <td><a href="http://brackets.io/?lang=en">Brackets</a>, <a href="https://atom.io/">Atom</a>, <a href="http://www.barebones.com/products/textwrangler/">TextWrangler</a>, <a href="http://smultron.sourceforge.net/">Smultron</a></td>
-       <td rowspan="2"> <a href="http://c9.io/">Cloud9 IDE</a>, <a href="http://jsbin.com/3/">JSBin</a></td>
-  </tr>
-  <tr><td>$$</td>
-       <td></td>
-       <td><a href="http://www.sublimetext.com/">SublimeText</a>, <a href="http://macromates.com/">TextMate</a>, <a href="http://www.panic.com/coda/">Coda</a>, <a href="http://macrabbit.com/espresso/">Espresso</a></td></tr>
-  </tbody>
-</table>
-
-<p>
-</p><center>
-<img style="height:200px" src="http://farm5.static.flickr.com/4096/4931311586_4f770d82e1.jpg">
-</center>
-<p></p>
-     </section>
+      <header><h1>Text Editors</h1></header>
+      <section class="content">
+        <br/>
+        <p>Many text editors provide special assistance for writing HTML and CSS, like syntax-highlighting and autocompletion.</p>
+        <ul>
+          <li>Sublime Text, TextMate (Mac)</li>
+          <li>Notepad++ (Windows)</li>
+          <li>JSBin, JSFiddle (Online)</li>
+        </ul>
+        <pre>
+          &lt;h1&gt;My JavaScript questions&lt;/h1&gt;
```

Comment:
might want to unindent these `<pre><code>` blocks. here's how they look now:
![screen shot 2016-07-11 at 9 25 58 am](https://cloud.githubusercontent.com/assets/2753545/16738197/910e3ce2-4749-11e6-9549-2cc603d7ed16.png)